iceoryx_posh/test/moduletests/test_mepoo_chunk_settings.cpp

```diff
@@ -0,0 +1,397 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "iceoryx_posh/mepoo/chunk_settings.hpp"
+
+#include "iceoryx_posh/mepoo/chunk_header.hpp"
+#include "test.hpp"
+
+namespace
+{
+using namespace ::testing;
+
+using iox::mepoo::ChunkHeader;
+using iox::mepoo::ChunkSettings;
+using PayloadOffset_t = iox::mepoo::ChunkHeader::PayloadOffset_t;
+
+// BEGIN GETTER METHOD TESTS
+
+TEST(ChunkSettings_test, payloadSizeReturnsCorrectValue)
```

Comment:
Nitpick: same here for all test cases, capital letter

Reply:
in this case it's the name of the method, so I thought it's better this way. Do you still want me to change it?

Reply:
We are not consistent in this way as of now and I do not know of any agreement. I favor lower case names for the same reason.
BTW: I like lower case for private API and upper case for public API. Subtle, easy on the eye, but noticeable (mainly interesting if public methods are called to implement private ones).

Reply:
I'd tend the other way. I'd rather rename the test case to something like `CallingPayloadSizeMethodReturnsCorrectValue` and keep all unit test cases upper case. But that's just what makes it easier for me to read, no hard feelings here.